.github/workflows/snapshot_release.yml

```diff
@@ -37,6 +37,11 @@ jobs:
           ref: ${{ github.event.pull_request.head.ref }}
           repository: ${{ github.event.pull_request.head.repo.full_name }}
 
+      - name: Set up secrets
+        run: |
+          echo "CYPRESS_RECORD_KEY=$(echo YzE3ZDU4OGItZTBkOC00ZjJmLTg4NjYtNzJmNmFmYmRhNGQxCg== | base64 -d)" >> $GITHUB_ENV
```

Comment:
@wetneb Shouldn't we also use GitHub encrypted secrets for Cypress key and Coveralls token?

Reply:
no, because then they would not be available to pull requests, where we need them.